scopes/scope/scope/get-scope-options.ts

```diff
@@ -0,0 +1,7 @@
+import { ComponentType } from 'react';
+import { ScopeModel } from '.';
+
+export type GetScopeOptions = {
```

Comment:
This is for ui specific. the name getScope is too generic, I wouldn't think it's UI specific. maybe rename it a bit (also the filename)